vividus-util/src/test/java/org/vividus/jackson/databind/PatternDeserializerTests.java

```diff
@@ -33,17 +33,21 @@
     void shouldCreatePattern() throws IOException
     {
         String regex = ".*";
-        JsonParser jsonParser = mock(JsonParser.class);
-        when(jsonParser.getText()).thenReturn(regex);
-        assertEquals(regex, new PatternDeserializer().deserialize(jsonParser, null).toString());
+        try (JsonParser jsonParser = mock(JsonParser.class))
```

Comment:
looks useless, does it make sense to suppress such PMD violations in these tests?

Reply:
Good idea!